editor/editor_property_name_processor.cpp

```diff
@@ -177,6 +185,7 @@ EditorPropertyNameProcessor::EditorPropertyNameProcessor() {
 	capitalize_string_remaps["msaa"] = "MSAA";
 	capitalize_string_remaps["nfc"] = "NFC";
 	capitalize_string_remaps["normalmap"] = "Normal Map";
+	capitalize_string_remaps["ofs"] = "Offset";
```

Comment:
This should be renamed as well, for the same reasons as above. It's confusing for it to be "Offset" in the inspector and "ofs" in scripts.